### DISCUSSION TITLE
Storage compatibility with FibreChannel

### DISCUSSION THREAD

**Discussion by leo79901** [apache/cloudstack#10005] [Q&A] [answered]
Hi,
 I'm testing ACS with KVM.
 I found that ACS compatibility with Fibre Channel storage exists for HPE and Pure. There is no suitable Fibre Channel storage mentioned for Dell. 
 Is there a plan to work with Dell? For example, Dell Unity, is so popular.

**Comment by rajujith** [comment, marked as answer]
@leo79901  You should be able to use any fiber channel storage as a shared mount point type in CloudStack. You need to find a suitable clustered filesystem OCFS2, GFS, etc. HPE Primera and Pure have some management plugins but if you are not looking for managed storage you should be able to use any.
